app/models/webhooks/slack/base.rb

```diff
@@ -20,13 +20,11 @@ def attachments
       text:        attachment_text,
       fields:      attachment_fields,
       fallback:    attachment_fallback,
-      color:       attachment_color
+      color:       attachment_color,
+      mrkdwn_in: [:text]
     }]
   end
 
-  def attachment_color
```

Comment:
I don't think we want to drop this from the base, otherwise we have to define it in each of the sub classes, which is repeating ourself, which is bad.